screeps_console/command.py

```diff
@@ -19,6 +18,11 @@ def __init__(self):
         self.listbox = False
         self.listwalker = False
         self.edit = False
+        settings = getSettings()
+        self.apiclient = ScreepsConnection(
+                u=settings['screeps_username'],
+                p=settings['screeps_password'],
+                ptr=settings['screeps_ptr'])
```

Comment:
Why not just call `getApiClient` here? That way the settings and connection code doesn't have to be defined twice, and if changes are made to the ScreepsConnection or how it gets created we only have to change a single spot.